packages/loaders/src/styled-elements.js

```diff
@@ -23,42 +23,62 @@ StyledCircle.propTypes = {
   transform: PropTypes.string
 };
 
-const COMPONENT_ID = 'loaders.dots';
+export const SpinnerCircle = styled.circle.attrs({
+  cx: 40,
+  cy: 40,
+  r: 34,
+  fill: 'none',
+  stroke: 'currentColor',
+  strokeLinecap: 'round',
+  strokeWidth: ({ strokeWidthValue }) => strokeWidthValue,
+  strokeDasharray: ({ dasharrayValue }) => `${dasharrayValue} 250`,
+  transform: props => props.transform
+})`
+  transform-origin: center center;
```

Comment:
Does it make sense to identify malleable attributes via `will-change` here? If not, then we should probably remove `will-change` from the Dots circle above.